docker_services_cli/env.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Docker-Services-CLI is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Environment module."""
+
+import os
+import sys
+from distutils.version import StrictVersion
+
+import click
+
+
+def _setdefault_env(variable, default_value):
+    """Set environmental variable value if it does not exist."""
+    os.environ[variable] = os.environ.get(variable, default_value)
+
+
+def _is_version(version):
+    """Checks if a string is a version of the format `x.y.z`.
+
+    NOTE: It is not mandatory to be up to patch level. The following would be accepted:
+    - 10.1
+    - 9
+    - 15.0.1a2
+    """
+    try:
+        # StrictVersion fails on plain numbers (e.g. "10")
+        if version.isnumeric():
+            return True
+        StrictVersion(version)
+        return True
+    except ValueError:
+        return False
+
+
+def _setversion_env(variable, default_value):
```

Comment:
renamed by `service_version` and `default_version`. Mainly because "by our default it would be a major, but if the user changes it, it is not a *default_major_version* but just a *default_version*"